std/variant.d

```diff
@@ -1234,6 +1234,14 @@ public:
     assert(a[5] == 7);
 }
 
+@system unittest
```

Comment:
It seems this should be `@safe`

Reply:
Addressed in 6553c2e
Thanks for the review :+1:

Reply:
No problems, thanks for the bug fix.